### PR TITLE
Missing dependencies & update commands to install

### PR DESCRIPTION
**Pull request by thimaisogilis**

### CODE REVIEW

File: languages/en/developer-guide/quick-start/run-tuleap.rst [hunk 1]
```diff
@@ -31,14 +33,15 @@ First start of Tuleap
...
+    $ npm install
     $ npm run build
     $ make dev-setup
     $ make start
     $ make post-checkout
 
 .. NOTE:: docker will download base images for mysql, tuleap, … Please be patient!
 
-Then you need to know the IP address of the web container, with ``docker inspect tuleap-web | grep '"IPAddress"'`` and
+Then you need to know the IP address of the web container, with ``docker inspect [$TULEAP_WEB_ID_DOCKER_CONTAINER] | grep '"IPAddress"'`` and
```
Comment (vaceletm) [inline languages/en/developer-guide/quick-start/run-tuleap.rst]: You might want to use `make show-ips` that will advertise ips of all running containers

File: languages/en/developer-guide/quick-start/run-tuleap.rst [hunk 2]
```diff
@@ -90,6 +95,12 @@ If you need to connect to the server you can run:
     $ docker exec -ti tuleap-web bash
     $> export TERM=linux
 
+or
```
Comment (vaceletm) [inline languages/en/developer-guide/quick-start/run-tuleap.rst]: I'll suggest to just talk about `make bash-web` to limit the amount of doc

File: languages/en/developer-guide/quick-start/run-tuleap.rst (hunk 1 above)
Comment (thimaisogilis) [reply languages/en/developer-guide/quick-start/run-tuleap.rst]: :ok:

File: languages/en/developer-guide/quick-start/run-tuleap.rst (hunk 2 above)
Comment (thimaisogilis) [reply languages/en/developer-guide/quick-start/run-tuleap.rst]: :ok:

**Pull request merged**